react-function-components/src/index.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+const div = document.querySelector('#root');
+
+const CustomButton = <button>Click Me!</button>;
+
+ReactDOM.render(CustomButton, div);
```

Comment:
This isn't mounting a `CustomButton` function component, this is mounting a `button` component:

---

![image](https://user-images.githubusercontent.com/16143771/145653438-e327657f-903b-4958-a5df-c648bea2acca.png)